packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md

```diff
@@ -0,0 +1,11 @@
+# Snapshot report for `test/fast-usdc/fast-usdc.test.ts`
+
+The actual snapshot is saved in `fast-usdc.test.ts.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## writes feed policy to vstorage
+
+> Snapshot 1
+
+    '{"blockHeight":"0","values":["{\\"chainPolicies\\":{\\"Arbitrum\\":{\\"cctpTokenMessengerAddress\\":\\"0x19330d10D9Cc8751218eaf51E8885D058642E08A\\",\\"chainId\\":42161,\\"confirmations\\":1,\\"nobleContractAddress\\":\\"0x19330d10D9Cc8751218eaf51E8885D058642E08A\\"}},\\"nobleAgoricChannelId\\":\\"channel-21\\",\\"nobleDomainId\\":4}"]}'
```

Comment:
This could be prettied up a bit using...

```
import { documentStorageSchema } from '@agoric/internal/src/storage-test-utils.js';
```

Reply:
Done